### PR TITLE
Week2 Assignments

### PR DESCRIPTION
**Pull request by PotzarellaMozarella**
Hey Imran! 

Here are the week 2 assignments:
- Page with 3 and 2 column layouts with a form
- Moving blocks using transition

### CODE REVIEW

**Review by imranmd — APPROVED** [review]

File: Week-2/Assignment1/index.html [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400&family=Lora:ital,wght@0,400;0,500;1,400;1,500;1,600&family=Open+Sans:ital,wght@0,300;0,400;0,500;1,300;1,400&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- For facebook/twitter sharing -->
```
Comment (imranmd) [inline Week-2/Assignment1/index.html:33]: Good

File: Week-2/Assignment1/index.html [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+          <select class="input-section">
+            <option value="India">India</option>
+            <option value="USA">USA</option>
+            <option value="UK">UK</option>
+            <option value="Pakistan">Pakistan</option>
+            <option value="Thailand">Thailand</option>
+            <option value="Japan">Japan</option>
+            <option value="Brazil">Brazil</option>
+          </select>
+          <br />
```
Comment (imranmd) [inline Week-2/Assignment1/index.html:365]: Try not to give breaks, See if this can addressed using CSS

File: Week-2/Assignment1/index.html [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+          <br />
+          <input
+            type="checkbox"
+            name="tnc"
+            required
+          /><label> I agree to the <a href="">terms and conditions</a></label>
+          <br />
```
Comment (imranmd) [inline Week-2/Assignment1/index.html:372]: Same goes here

File: Week-2/Assignment1/index.html [hunk 4]
```diff
@@ -0,0 +1,391 @@
...
+                    voluptates asperiores! Natus, ratione eaque.
+                  </p>
+                </summary>
+                <a href="">Click here</a>
+              </article>
+            </div>
+            <div>
+              <article class="article-small">
+                <h2 class="article-title">Lorem ipsum</h2>
+                <summary>
```
Comment (imranmd) [inline Week-2/Assignment1/index.html:198]: Not sure if summary tag works in all browsers , like old IE7/IE8

File: Week-2/Assignment1/index.html [hunk 5]
```diff
@@ -0,0 +1,391 @@
...
+          <br /><br />
+          <label> Firstname* </label>
+          <input class="input-section"
+            type="text"
+            name="firstname"
+            size="15"
+            placeholder="Your name"
+            required
+          />
+          <br />
```
Comment (imranmd) [inline Week-2/Assignment1/index.html:256]: Too many breaks, Try to avoid 

Best practice is to use CSS

File: Week-2/Assignment1/index.html [hunk 6]
```diff
@@ -0,0 +1,391 @@
...
+          <input
+            type="checkbox"
+            name="Fashion"
+          />
+          Fashion<br />
+          <input
+            type="checkbox"
+            name="Real Estate"
+          />
+          Real Estate<br />
```
Comment (imranmd) [inline Week-2/Assignment1/index.html:303]: Indentation

File: Week-2/Assignment1/index.html [hunk 7]
```diff
@@ -0,0 +1,391 @@
...
+            name="phone"
+            size="10"
+            placeholder="9009009000"
+            required
+          />
+          <br />
+          <br />
+          Address
+          <br />
+          <textarea class="input-section"
```
Comment (imranmd) [inline Week-2/Assignment1/index.html:317]: Use some formatting or linting tools or extension in VS Code

File: Week-2/Assignment2/index.html [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+      property="og:title"
+      content="Playground | Season 2"
+    />
+    <meta
+      property="og:type"
+      content="video.series"
+    />
+    <meta
+      property="og:url"
+      content="https://www.youtube.com/c/PLAYGROUND_GLOBAL"
```
Comment (imranmd) [inline Week-2/Assignment2/index.html:24]: I hope you know what these tags do :)

File: Week-2/Assignment2/index.html [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+        <div class="box-red"></div>
+        <div class="box-yellow"></div>
+      </div>
+
+      <div class="row">
+        <div class="box-blue"></div>
+        <div class="box-green"></div>
+      </div>
+      <!--Text as block behind boxes-->
+      <div class="hi">Hi!</div>
```
Comment (imranmd) [inline Week-2/Assignment2/index.html:63]: Is this 'HI' displays everytime or only on hover?

**Review by imranmd — APPROVED** [review]

File: Week-2/Assignment2/index.html (hunk 9 above)
Comment (PotzarellaMozarella) [reply Week-2/Assignment2/index.html:63]: Its hidden behin the blocks using z-index, appears only on hover.

File: Week-2/Assignment1/index.html (hunk 7 above)
Comment (PotzarellaMozarella) [reply Week-2/Assignment1/index.html]: im using prettier but on formatting it indents like this, will try o figure out how to improve this

File: Week-2/Assignment1/index.html (hunk 5 above)
Comment (PotzarellaMozarella) [reply Week-2/Assignment1/index.html]: Made the necessary changes to remove br

File: Week-2/Assignment1/index.html (hunk 2 above)
Comment (PotzarellaMozarella) [reply Week-2/Assignment1/index.html]: made changes

File: Week-2/Assignment1/index.html (hunk 4 above)
Comment (imranmd) [reply Week-2/Assignment1/index.html:160]: We should check HTML documentation to validate this

**Review by imranmd — APPROVED** [review]
Approved

**Pull request merged**